src/fatxfs_fuse.c

```diff
@@ -619,17 +612,22 @@ void fatx_fuse_print_usage(void)
     fprintf(stderr, "Usage: %s <device> <mountpoint> [<options>]\n", prog_short_name);
     fprintf(stderr, "   or: %s <device> <mountpoint> --drive=c|e|x|y|z [<options>]\n", prog_short_name);
     fprintf(stderr, "   or: %s <device> <mountpoint> --offset=<offset> --size=<size> [<options>]\n\n", prog_short_name);
-    fprintf(stderr, "General Options:\n"
-                    "    -o opt, [opt...]       mount options\n"
-                    "    -h --help              print help\n"
-                    "    -V --version           print version\n\n"
+    fprintf(stderr, "General options:\n"
+                    "    -o opt, [opt...]               mount options\n"
+                    "    -h --help                      print help\n"
+                    "    -V --version                   print version\n\n"
                     "FATXFS options:\n"
-                    "    --drive=<letter>       mount a partition by its drive letter\n"
-                    "    --offset=<offset>      specify the offset (in bytes) of a partition manually\n"
-                    "    --size=<size>          specify the size (in bytes) of a partition manually\n"
-                    "    --sector-size=<size>   specify the size (in bytes) of a device sector (default is 512)\n"
-                    "    --log=<log path>       enable fatxfs logging\n"
-                    "    --loglevel=<level>     control the log output level (a higher value yields more output)\n\n");
+                    "    --drive=<letter>               mount a partition by its drive letter\n"
+                    "    --offset=<offset>              specify the offset (in bytes) of a partition manually\n"
+                    "    --size=<size>                  specify the size (in bytes) of a partition manually\n"
+                    "    --sector-size=<size>           specify the size (in bytes) of a device sector (default is 512)\n"
+                    "    --log=<log path>               enable fatxfs logging\n"
+                    "    --loglevel=<level>             control the log output level (a higher value yields more output)\n\n"
+                    "Disk formatting options:\n"
+                    "    --format=<format>              specify the format (retail, f-takes-all) to initialize the device to\n"
+                    "    --sector-size=<size>           specify the size (in bytes) of a device sector (default is 512)\n"
+                    "    --sectors-per-cluster=<size>   specify the sectors per cluster when initializing non-retail partitions (default is 128)\n"
+                    "    --destroy-all-existing-data    acknowledge that device formatting will destroy all existing data\n\n");
```

Comment:
`--destroy-all-existing-data` is simply a safety feature designed to try and force users to acknowledge the consequences of what they're trying to do. It is mandatory (currently) to include this flag if also using `--format`